src/main/java/org/generationcp/middleware/domain/inventory/manager/TransactionsSearchDto.java

```diff
@@ -17,7 +17,8 @@
 	private String stockId;
 	private List<Integer> transactionIds;
 	private String createdByUsername;
-	private String transactionType;
+	private Integer transactionType;
```

Comment:
@vmaletta transactionTypes and transactionStatus should be a List<Integer> to allow search by multiple values.